ratelimit_test.go

```diff
@@ -125,6 +126,44 @@ var takeTests = []struct {
 	}},
 }}
 
+var availTests = []struct {
+	capacity     int64
+	fillInterval time.Duration
+	take         int64
+	sleep        time.Duration
+
+	expectCountAfterTake  int64
+	expectCountAfterSleep int64
+}{{ // should fill
```

Comment:
let's make this comment into an "about" field for consistency with
other table-driven tests, and log it. (it means that when a test
does fail, we can quickly find out which one has failed)